### ISSUE TITLE
Mod not working Ish.

### ISSUE THREAD

**Issue by CREEPERDUDE062** [19miffyliu/Kill-It-With-Shovel#1]
It only disables it for half a second and then comes back even when I change the config. I also have berserk mode off. So, then I'm stuck in a loop of hitting it to not get shot.

**Comment by 19miffyliu** [comment]
Weird. Can you give me a video demonstration or a screenshot of the config file? Also, did turning beserk mode off actually do anything or did it just went beserk as normal?

**Comment by CREEPERDUDE062** [comment]
oh i didn't know you responded but the berserk mode did not activate when i hit it so that worked but i got a funny video

https://github.com/19miffyliu/Kill-It-With-Shovel/assets/155585089/1ab5a281-8ae1-47fb-91c3-bba7a12cd3ad

.

**Comment by 19miffyliu** [comment]
Thank you hahahahaha you made my day (dying from laughing too hard). But seriously, try sending me a screenshot of you and your friend's config settings and/or mod lists. I can't fix things if I don't know what is wrong. It seems to me that everything is working fine except what the disabled duration is too short.

Also I uploaded a 1.0.1 version which fixes a bug when you hit the turret in berserker mod. Try updating the mod and see if it works?

**Comment by CREEPERDUDE062** [comment]
im using mod manager.

![Screenshot 2024-01-03 215751](https://github.com/19miffyliu/Kill-It-With-Shovel/assets/155585089/23dadae9-fa41-4496-a424-cb94720b0c13)

**Comment by 19miffyliu** [comment]
Hmmm...that looks alright. The only problem I can think of is the mod conflict. Did you have any mod that also tweaks the turret functions like FairAI or RemoteRemastered?

**Comment by CREEPERDUDE062** [comment]
Fair Ai. Well that sucks.

**Comment by 19miffyliu** [comment]
Yeah I thought so. Fair Ai tweaks a lot of turret behaviors which makes it incompatible with mine. It's a great mod...but I'm afraid I cannot solve this mod conflict issue. Thanks for telling me though, I will address this problem in the description. 

From what I observed in your video, my mod does work (AKA the turret can be disabled with shovel). I think you can still use it with Fair AI, but know that the duration will be half a second no matter the config.

Oh and I'm so going to add that video to description hahahaha that is so funny
